### DISCUSSION TITLE
What curb load zone metrics are right for you?

### DISCUSSION THREAD

**Discussion by bhamlinSDOT** [openmobilityfoundation/curb-data-specification#7] [Policy]
Hi all, 

Jumping off from our meeting this morning, I wanted to open up a conversation to further discuss **load zone metrics** that working group members find necessary (or wish they had access to) in order to answer their policy or usage questions/requirements. Having a common understanding of need from working group members will help the steering committee to develop the Metrics API/Methodology Standard into something that will be useful for all. 

**Questions we had:** 
1. What Metrics are useful for **cities** in curb areas (turnover…)? 
2. What Metrics are useful for **commercial** users of the curb areas? 
3. How do you define **occupancy** (small vs large vehicles, distance used, spaces utilized, demarcation)? 
4. How do you **aggregate** sensors and events towards one standard curb usage value? 
5. What questions or actions are your trying to solve/perform with these metrics?
6. Anything else? 

 We would really appreciate your thoughts on this!

**Comment by jfh01** [comment]
Here are some curb performance metrics from a paper ("Curbside Management Challenges and Opportunities: Insights from the Public and Private Sectors in 14 US Cities") from the Urban Freight Lab team. Sharing here with permission from one of the authors, Dr. Ranjbari.

[Curb Challenges - Metrics.pdf](https://github.com/openmobilityfoundation/curb-data-specification/files/6749747/Curb.Challenges.-.Metrics.pdf)

**Comment by akshay-malik** [comment]
Hi All, 
The list Jascha shared is much better laid out and covers many more dimensions than what I had to share. But anyways, in Philly we are working to set up a pilot that would track the following metrics to begin with:

- Bookings
- Revenue
- Utilization (% of available space consumed by authorized loading activities at any given time)
- Dwell time
- Percent full time (how often a driver will arrive at a zone and be unable to book it because it's already full)
- Unauthorized uses reported
- Enforcement scans, grouped by outcome (unauthorized/authorized/overstayed/not yet arrived)

We hope to edit this list as we learn more from other cities (through OMF's working group), local delivery operators and other stakeholders.

**Comment by jacobmalleau** [comment]
Some comments on questions 3 and 4.
I think it makes the most sense to complete occupancy on a per space basis. In the vast majority of cases, occupancy will be obtained from sensors which are located in delineated spaces and therefore provides a natural way to divide up a curb segment. 
From there, I think occupancy calculations should be kept simple: amount of time space is occupied / total amount of time * 100% where total amount of time can be dynamic (15 minutes, an hour, the length of the time the regulation applies, etc.) with the numerator updated accordingly. If the rate needs to be standardized I would do say an hourly basis makes the most sense. This calculation can be done for 1 space, or several spaces over a given regulation zone. This also helps simplify how event/sensor data can be aggregated - it can just be used to define whether a space is occupied or not.

**Reply by nick-jonas** [reply]
In speaking for our on-ground sensors ([Pebble](https://www.sidewalklabs.com/products/pebble)), we are preparing to accommodate for cities that have non-demarcated curb parking space.  

From what I understand, the majority of _paid_ parking spaces (with meters) in cities are demarcated, with cities like New York City and Seattle being big exceptions.  Free parking spaces, which makes up the majority of parking in all cities are non-demarcated.  

Interesting sidenote, from a member on my team:

> There was a claim by NYC that non-demarcated spaces yields MORE spaces at the curb, but that's not proven.  It really depends on how people park.  In residential streets with high demand, cars seem to shuffle around and end up parked really close to each other.  In metered spaces, maybe because drivers paid for the space, they tend to leave big gaps between vehicles when there's no markings on the pavement.

**Comment by StephenHDDOT** [comment]
Hey Brian, some comments here compiled from DDOT:

- Number of vehicles stopping in zone
- Authorized
- Unauthorized (non-commercial, unpaid, overstay)
- ​type of vehicle
- length of available curb at zone (if more than one vehicle can load in zone) / Utilization (% of available space consumed by authorized loading activities at any given time)
- type of activity
- duration / dwell time
- Whether use was paid / unpaid, and revenue generated
- Citations given by reason

And not a specific metric, but some interesting/valuable information would be how do curb users think of loading zones/curb access?

**Comment by tomcarr178** [comment]
Curb policy: Metered vs. free, type of zone, time limits, restrictions, etc.
Dwell time and occupancy: not only the duration but the arrival and departure time.  
Type of vehicle:  If possible, car, truck, taxi, TNC, etc. and if we can distinguish ZEV vs. gas
Paid/unpaid parking: Rate, payment data. Be able to identify disabled placard use vs. violation for unpaid parking.
